syncer/heartbeat.go

```diff
@@ -122,12 +123,12 @@ func (h *Heartbeat) AddTask(name string) error {
 	if h.primary == nil {
 		// open DB
 		dbCfg := h.cfg.primaryCfg
-		dbDSN := fmt.Sprintf("%s:%s@tcp(%s:%d)/?charset=utf8&interpolateParams=true&readTimeout=1m", dbCfg.User, dbCfg.Password, dbCfg.Host, dbCfg.Port)
-		primary, err := sql.Open("mysql", dbDSN)
+		dbCfg.RawDBCfg.ReadTimeout = "1m"
```

Comment:
```suggestion
		if dbCfg.RawDBCfg != nil {
			dbCfg.RawDBCfg.SetReadTimeout("1m")
		}
```

Reply:
I think we could init a pointer to `RawDBCfg`

Reply:
ok